src/scripts/shared/use-duration.ts

```diff
@@ -12,7 +12,7 @@ export const useDuration = (start: string) => {
 
   const updateDuration = React.useCallback(() => {
     setDuration(timeElapsed(start));
-  }, [setDuration]);
+  }, [duration]);
```

Comment:
```suggestion
  }, [setDuration, start]);
```
Can you verify with this?